packages/builder/src/components/design/Panel.svelte

```diff
@@ -76,9 +80,12 @@
     justify-content: space-between;
     align-items: center;
     padding: 0 var(--spacing-l);
-    border-bottom: var(--border-light);
     gap: var(--spacing-l);
   }
+  .header-wrap {
+    border-bottom: var(--border-light);
+    padding: var(--spacing-l) 0px;
```

Comment:
This is adding additional padding into the core Panel component, so will affect the layout of all other panels - could we remove this padding and instead add it to node we insert inside this slot, so it doesn't affect all panels?

I know it probably doesn't affect layout because the padding is probably less than what spacing is already there due to the fixed flex rule, but I'd prefer to leave it up to whatever gets rendered inside the slot, so it's always in control.

Reply:
Yep, that makes sense, I'll make the change here and ensure that it can't interfere with the padding of the component in general. As for renaming the `General` tab, I think you're right in that it gives a little bit more context.